archai/nas/model_desc_builder.py

```diff
@@ -409,6 +409,10 @@ def build_model_stems(self, in_shapes:TensorShapesList,
                           in_len=1, trainables=None) \
                 for op_name in ops]
 
+        #stems = [OpDesc(name=op_name, params={'conv': conv_params},

```

Comment:
Instead of commenting this out, you can use identity as stem op. We cannot push these kind of randomly commented out code in master because it will impact other algorithms such as DARTS which will stop working. If you don't want certain piece of code to execute, try to see what options you use in YAML or which ops are "no op" or add a switch that you can turn on or off in YAML.